spec/rspec/core/reporter_spec.rb

```diff
@@ -38,6 +38,16 @@ module RSpec::Core
     describe 'start' do
       before { config.start_time = start_time }
 
+      it 'notifies formatters of the seed used' do
+        formatter = double("formatter")
+        reporter.register_listener formatter, :seed
+
+        expect(formatter).to receive(:seed).with(
+          an_object_having_attributes(:seed => config.seed, :seed_used? => config.seed_used?)
+        )
+        reporter.start 1
+      end
```

Comment:
Yeah I think I'd prefer you move this back below and just add the order expectation